capsules/extra/src/net/ipv6/ipv6_send.rs

```diff
@@ -107,7 +109,7 @@ pub struct IP6SendStruct<'a, A: time::Alarm<'a>> {
     tx_buf: TakeCell<'static, [u8]>,
     sixlowpan: TxState<'a>,
     radio: &'a dyn MacDevice<'a>,
-    dst_mac_addr: MacAddress,
+    dst_mac_addr: Cell<MacAddress>,
```

Comment:
Why not an OptionalCell instead of having some default?

Reply:
I agree that an OptionalCell would be better here. I originally added these defaults because the kernel/capsules had some hard coded values (used for libtock-c examples). This shouldn't be an issue and I'll submit a PR updating the libtock-c examples.